modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -645,6 +662,32 @@ public Page<Object> getRelatedSystemObjectEntries(
 				serviceBuilderObjectEntry.getPrimaryKey()));
 	}
 
+	@Override
+	public Object getSystemBaseModel(
```

Comment:
Hi @MarinhoFeliphe, I'm thinking about a better name for this method. 
I'm thinking about change to `getSystemObjectData`, what do you think?

Let me know if you have any suggestions, @DaneszLR